pages/message-sent/index.tsx

```diff
@@ -34,7 +40,10 @@ export default function MessageSent() {
                             <text className="font-inter font-normal">🕵️‍♀️ Get your own incognito messages! </text>
                             <text className="font-inter font-normal">🔔 See what’s new in your school or community</text>
                         </div>
-                        <button onClick={() => router.push(publicRuntimeConfig.DOWNLOAD_BETTERSOCIAL_APP_URL)} className=" w-full bg-cyan hover:bg-cyan10 text-white font-lato flex py-2 px-4 rounded-lg justify-center items-center gap-2 text-justify">
+                        <button onClick={onGoToProfileButtonClicked} className=" w-full bg-white hover:bg-cyan10 text-cyan hover:text-white border-cyan border-2 font-lato flex py-2 px-4 rounded-lg justify-center items-center gap-2 text-justify mb-2">
+                            {`Back to ${localStorage.getItem(MessageEnum.targetUserName)}'s profile`}
+                        </button>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this code assumes that `localStorage.getItem(MessageEnum.targetUserName)` will always return a valid username. If it doesn't, the button text will be "Back to 's profile", which might confuse users. Consider adding a default value or error handling here as well.

```diff
- {`Back to ${localStorage.getItem(MessageEnum.targetUserName)}'s profile`}
+ {`Back to ${localStorage.getItem(MessageEnum.targetUserName) || 'User'}'s profile`}
```
This will display "Back to User's profile" if the username is not found in local storage.


<!-- This is an auto-generated comment by OSS CodeRabbit -->